src/cs50/__init__.py

```diff
@@ -1,34 +1,48 @@
+import os
 import sys
 
-from .cs50 import eprint, get_char, get_float, get_int, get_string
 try:
-    from .cs50 import get_long
-except:
-    pass
 
-from . import flask
+    # Save student's sys.path
+    path = sys.path[:]
 
+    # In case student has files that shadow packages
+    sys.path = [p for p in sys.path if p not in ("", os.getcwd())]
 
-class CustomImporter(object):
-    """
-    Import cs50.SQL lazily so that rest of library can be used without SQLAlchemy installed.
+    # Import cs50_*
+    from .cs50 import eprint, get_char, get_float, get_int, get_string
+    try:
+        from .cs50 import get_long
+    except:
+        pass
 
-    https://docs.python.org/3/library/imp.html
-    http://xion.org.pl/2012/05/06/hacking-python-imports/
-    http://dangerontheranger.blogspot.com/2012/07/how-to-use-sysmetapath-with-python.html
-    """
+    # Replace flask's logger
+    from . import flask
 
-    def find_module(self, fullname, path=None):
-        if fullname == "cs50.SQL":
-            return self
-        return None
+    class CustomImporter(object):
```

Comment:
Aesthetically we could maybe move this above the try block (so it isn't indented an extra level), but not a big deal

Reply:
Not opposed.